### DISCUSSION TITLE
Changing control panel colors

### DISCUSSION THREAD

**Discussion by vJoeyz** [statamic/cms#10921] [Troubleshooting] [answered]
There is a section about using custom CSS on the [page](https://statamic.dev/white-labeling) on white-labeling Statamic. I was wondering if there is an easy way to override the (tailwind) theme colors the control panel uses (the default blue)?

**Comment by ryanmitchell** [comment, marked as answer]
https://github.com/JayGeorge/admin-styler might get you there?

**Reply by vJoeyz** [reply]
Awesome, thanks.
